test/PC_D/LocalPreferences.toml

```diff
@@ -0,0 +1 @@
+[PC_D]
```

Comment:
What's this about?

Reply:
Oh, the test suite creates this file and it is not gitignored so I accidentally commited it.